app/controllers/admin_controller.rb

```diff
@@ -32,7 +32,7 @@ def reject
   end
 
   def new_members
-    @new_members = User.new_members
+    @new_members = User.new_members.reorder(:last_stripe_charge_succeeded)
```

Comment:
Very cool! I didn't know this existed. :)
I am going to test this locally to make sure it works as expected with nulls and then I will approve. https://apidock.com/rails/ActiveRecord/QueryMethods/reorder

Reply:
@RajivSah I know this change is maybe too small for a test but I like tests and it would save me time if there were one. This repo is very welcoming to tests :)

Reply:
lgtm
`User.new_members.reorder(:last_stripe_charge_succeeded).map(&:id) # => [1, 2, 3, 6, 10]` in my local after running `bundle exec rake populate:users`